config/game_options.txt

```diff
@@ -591,6 +591,10 @@ BOX_RANDOM_ENGINE Box Singulo,3
 BOX_RANDOM_ENGINE Box SM 1x3,1
 BOX_RANDOM_ENGINE Box SM 5x5,1
 BOX_RANDOM_ENGINE Box SM 3x,0
+BOX_RANDOM_ENGINE Box TEG,3
+BOX_RANDOM_ENGINE Box Empty,0
+BOX_RANDOM_ENGINE Box Antimatter,1
```

Comment:
The antimatter engine is old, not sure if functional, and tgstation removed it already.

Reply:
I know it work, We've used it multiple times on the server.